### DISCUSSION TITLE
The Latest Version og grape js V0.21.8 is not supporting ?

### DISCUSSION THREAD

**Discussion by SAKTHIVELMARIMUTHU** [GrapesJS/grapesjs#5712] [Q&A]
The Latest Version of grape js V0.21.8 is not supporting for my angular project . I am currently using the node version of v16.16.0 and the angular version is v11.0.5. Is there any way to fix this ?

**Comment by artf** [comment]
It's hard to help if you don't say what is the actual issue

**Reply by SAKTHIVELMARIMUTHU** [reply]
Is it possible to to add the component settings inside the Style Manager ? 
I am using this grape js version - 0.16.18
